src/fossa_cli.rs

```diff
@@ -143,22 +233,22 @@ async fn download(config_dir: &PathBuf) -> Result<PathBuf, Error> {
 // https://github.com/fossas/fossa-cli/releases/download/v3.7.2/fossa_3.7.2_darwin_amd64.zip
 // https://github.com/fossas/fossa-cli/releases/download/v3.7.2/fossa_3.7.2_linux_amd64.zip
 #[cfg(target_os = "windows")]
-fn download_url(version: &str) -> String {
+fn download_url(version: String) -> String {
```

Comment:
Generally functions should try to use references by default unless the reference becomes unwieldy (due to lifetimes). References are a lot easier to work with in general and don't require cloning everywhere.

Please change these functions back to `&str` (and same with `download`, `download_from_github`, etc).
If there's some reason this can't work or you think it's better this way though let's talk!